.gitignore

```diff
@@ -21,3 +21,5 @@
 
 # virtual machine crash logs, see http://www.java.com/en/download/help/error_hotspot.xml
 hs_err_pid*
+/target/
```

Comment:
I'd duplicate the git ignore from https://github.com/CUTR-at-USF/muser-firebase-export/blob/master/.gitignore, as it looks like the .iml files aren't being ignored.